ASan and TSan jobs do not appear to be running any Python tests

https://github.com/jiridanek/qpid-proton/runs/2569479535#step:14:17

```
Run ctest -C ${BuildType} -V -T Test --no-compress-output -R python-test 
  ctest -C ${BuildType} -V -T Test --no-compress-output -R python-test 
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    BuildType: 
    BuildDir: /home/runner/work/qpid-proton/qpid-proton/BLD
    InstallPrefix: /home/runner/work/qpid-proton/qpid-proton/INSTALL
    PKG_CONFIG_PATH: 
    VCPKG_DEFAULT_TRIPLET: x64-windows
    pythonLocation: /opt/hostedtoolcache/Python/3.6.13/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.6.13/x64/lib
   Site: fv-az214-850
   Build name: Linux-cc
Create new tag: 20210512-1906 - Experimental
Test project /home/runner/work/qpid-proton/qpid-proton/BLD
    Start 11: python-test
1/1 Test #11: python-test ......................***Failed  Required regular expression not found. Regex=[Ran .+ tests in .*

OK
]  1.04 sec

0% tests passed, 1 tests failed out of 1

Total Test time (real) =   1.14 sec

The following tests FAILED:
	 11 - python-test (Failed)
Errors while running CTest
Output from these tests are in: /home/runner/work/qpid-proton/qpid-proton/BLD/Testing/Temporary/LastTest.log
Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely.
Error: Process completed with exit code 8.
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -75,7 +75,7 @@ jobs:
     - id: ctest
       name: ctest
       working-directory: ${{env.BuildDir}}
-      run: ctest -VV --no-compress-output -R python-test ${{matrix.ctest_extra}}
+      run: ctest -C ${BuildType} -VV --no-compress-output -R python-test ${{matrix.ctest_extra}}
       shell: bash
     - name: Upload Test results
       if: always() && (steps.ctest.outcome == 'failure' || steps.ctest.outcome == 'success')

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -75,7 +75,7 @@ jobs:
     - id: ctest
       name: ctest
       working-directory: ${{env.BuildDir}}
-      run: ctest -C ${BuildType} -VV --no-compress-output -R python-test ${{matrix.ctest_extra}}
+      run: ctest -C ${BuildType} -VV --output-on-failure --no-compress-output -R python-test ${{matrix.ctest_extra}}
       shell: bash
     - name: Upload Test results
       if: always() && (steps.ctest.outcome == 'failure' || steps.ctest.outcome == 'success')

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -75,7 +75,7 @@ jobs:
     - id: ctest
       name: ctest
       working-directory: ${{env.BuildDir}}
-      run: ctest -C ${BuildType} -V -T Test --no-compress-output -R python-test ${{matrix.ctest_extra}}
+      run: ctest -VV --no-compress-output -R python-test ${{matrix.ctest_extra}}
       shell: bash
     - name: Upload Test results
       if: always() && (steps.ctest.outcome == 'failure' || steps.ctest.outcome == 'success')

--- a/tests/RuntimeCheck.cmake
+++ b/tests/RuntimeCheck.cmake
@@ -104,7 +104,7 @@ elseif(RUNTIME_CHECK STREQUAL "asan")
   set(SANITIZE_FLAGS "-g -fno-omit-frame-pointer ${CLANG_ASAN_FLAG} -fsanitize=address,undefined -fsanitize-recover=vptr")
   set(TEST_WRAP_PREFIX "${CMAKE_SOURCE_DIR}/tests/preload_asan.sh $<TARGET_FILE:qpid-proton-core>")
   list(APPEND TEST_ENV "UBSAN_OPTIONS=suppressions=${CMAKE_SOURCE_DIR}/tests/ubsan.supp")
-  list(APPEND TEST_ENV "LSAN_OPTIONS=suppressions=${CMAKE_SOURCE_DIR}/tests/lsan.supp")
+  list(APPEND TEST_ENV "LSAN_OPTIONS=suppressions=${CMAKE_SOURCE_DIR}/tests/lsan.supp:verbosity=1:log_threads=1")
 
 elseif(RUNTIME_CHECK STREQUAL "tsan")
   assert_has_sanitizers()

